### DISCUSSION TITLE
Extract formatting with texts inside a table

### DISCUSSION THREAD

**Discussion by jimmyzzxhlh** [jsvine/pdfplumber#1123] [Q&A]
Hi,

I'm currently using pdfplumber to extract texts from a table using `extract_tables`, but the format will be lost as only the text string is returned.

Is it possible to return the format? e.g. font/italic/bold, etc. 

Example PDF: https://www.humana-medicare.com/BenefitSummary/2024PDFs/H0028014000EOC24.pdf
Starting from page 49 there is a long table across multiple pages. I'd like to know specifically which text are bolded / underlined, etc. 

I know the font/size can be returned from `extract_words` API, but not sure how to do that in a table context.

**Comment by jsvine** [comment]
Does the approach described here achieve what you're looking to do?: https://github.com/jsvine/pdfplumber/discussions/1106#discussioncomment-8753886
